docs/distributed/README.md

```diff
@@ -55,11 +58,23 @@ minio server http://192.168.1.11/export1 http://192.168.1.12/export2 \
                http://192.168.1.15/export5 http://192.168.1.16/export6 \
                http://192.168.1.17/export7 http://192.168.1.18/export8
 ```
+#### Windows 
+
+```cmd
+set MINIO_ACCESS_KEY=<ACCESS_KEY>
+set MINIO_SECRET_KEY=<SECRET_KEY>
+minio.exe server http://192.168.1.11/C:\data http://192.168.1.12/C:\data ^
+                  http://192.168.1.13/C:\data http://192.168.1.14/C:\data ^
+                  http://192.168.1.15/C:\data http://192.168.1.16/C:\data ^
+                  http://192.168.1.17/C:\data http://192.168.1.18/C:\data
```

Comment:
I feel the mixed path style looks weird. However `/` should work instead of `\` ie I prefer
```
minio.exe server http://192.168.1.11/C:/data http://192.168.1.12/C:/data ^
                  http://192.168.1.13/C:/data http://192.168.1.14/C:/data ^
                  http://192.168.1.15/C:/data http://192.168.1.16/C:/data ^
                  http://192.168.1.17/C:/data http://192.168.1.18/C:/data
```

MS Windows works with both `/` and `\`